### PR TITLE
Refactor storage installation guides on Linux

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

**Comment by Sherlock113** [comment]
The original storage configuration guide is too general and thus removed. It is divided into different separate documents to give detailed installation guides to readers. The Ceph guide will be written in detail later as the configuration of the server is a little bit complicated. And the Longhorn guide will be added later as well.  @stoneshi-yunify @FeynmanZhou This is a very large PR. Take your time. Thanks.

File: content/en/docs/installing-on-linux/persistent-storage-configurations/install-ceph-csi-rbd.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+description: 'How to create a KubeSphere create with Ceph providing storage services.'
+linkTitle: "Install Ceph"
+weight: 3350
+---
+
+With a Ceph server, you can choose [Ceph RBD](https://kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd) or [Ceph CSI](https://github.com/ceph/ceph-csi) as the underlying storage plugin. Ceph RBD is an in-tree storage plugin on Kubernetes, and Ceph CSI is a Container Storage Interface (CSI) driver for RBD, CephFS. 
+
+###  Which plugin to select for Ceph 
+
+Ceph CSI RBD is the preferred choice if you work with **14.0.0 (Nautilus)+** Ceph cluster. Here are some reasons:
```
Comment (stoneshi-yunify) [inline content/en/docs/installing-on-linux/persistent-storage-configurations/install-ceph-csi-rbd.md:13]: 14.0.0 and later

File: content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+2. Create a separate configuration file for your add-on to list all the necessary parameters and provide the path of the file in `config-sample.yaml` so that KubeKey can reference it during installation.
+
+For more information, see [add-ons](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md).
+
+## Default Storage Class
+
+KubeKey supports the installation of different storage plugins and storage classes. No matter what storage systems you will be installing, you can specify whether it is a default storage class in its configuration file. If KubeKey detects that no default storage class is specified, it will install [OpenEBS](https://github.com/openebs/openebs) by default.
+
+OpenEBS Dynamic Local PV provisioner can create Kubernetes Local Persistent Volumes using a unique HostPath (directory) on the node to persist data. It is very convenient for users to get started with KubeSphere when they have no specific storage system. However, in production, it is highly recommended that you install your own persistent storage and configure the corresponding storage plugin and storage class to dynamically provision volumes.
```
Comment (stoneshi-yunify) [inline content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md:41]: `However, in production, it is highly recommended that you install your own persistent storage and configure the corresponding storage plugin and storage class to dynamically provision volumes.` , this sentence should be removed. There are many adopters of openebs using local pv in production, see https://github.com/openebs/openebs/blob/master/ADOPTERS.md.

File: content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Multi-storage Solutions
+
+If you intend to install more than one storage plugins, only set one of them as the default one with the storage class name you want KubeSphere to use. Otherwise, KubeKey will be confused about which storage class to use.
```
Comment (stoneshi-yunify) [inline content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md:45]: `only set one of them as the default one with the storage class name you want KubeSphere to use.` sounds tedious. Maybe just `only one could be set as the default storage class for KubeSphere to use.`

File: content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md (hunk 2 above)
Comment (Sherlock113) [reply content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md]: Changed.

File: content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md (hunk 3 above)
Comment (Sherlock113) [reply content/en/docs/installing-on-linux/persistent-storage-configurations/understand-persistent-storage.md]: Changed.

**Pull request merged**